index.js

```diff
@@ -22,9 +22,65 @@ const copyWithTemplate = async (from, to, variables) => {
 	await writeFile(to, generatedSource);
 };
 
-const fromPath = file => path.join(__dirname, 'template', file);
+const useTsx = process.argv.includes('--tsx');
+let templatePath = 'templates/js';
+
+if (useTsx) {
+	templatePath = 'templates/tsx';
```

Comment:
```suggestion
	templatePath = 'templates/ts';
```

Could you rename the folder too?